GameData/SPEngine/Parts/LR87_proc.cfg

```diff
@@ -18,42 +18,19 @@
 	}
 	@identicalParts = SPEngineLR87,SPEngineLR87Twin
 	!engineType = // prevent ROE from deleting us
-	@MODULE[ModuleEngines*]
-	{
-		%powerEffectName = Hypergolic-Lower
-	}
-	@PLUME
-	{
-		@name = Hypergolic-Lower
-	}
 }
 
-+PART[ROE-LR87]:NEEDS[ROEngines]:BEFORE[ZREALPLUME]
++PART[ROE-LR87]:NEEDS[ROEngines]:FOR[SPEngine]
 {
 	@name = SPEngineLR87Twin
 	@title = T-class Procedural Engine
 	@description = Hypergolic atmospheric engine, like LR87 family (Titan II and later). <b><color=green>From ROEngines mod</color></b>
 	@TechRequired = orbitalRocketry1962
-	@node_stack_top[1] *= 0.7
-	@node_stack_bottom[1] *= 0.7
-	@MODEL
-	{
-		// Half the thrust of original part, so let's halve the exit plane area
-		%scale = 0.7, 0.7, 0.7
-	}
```

Comment:
Why is this being removed?

To explain why it's there in the first place: the T-class engine has a thrust rating roughly equal to _one_ chamber of an LR87.  (You're meant to cluster two of them to make a standard LR87.)  So a two-chamber model represents an engine that looks like an LR87 but has _half_ the total thrust, thus should be smaller.

Reply:
do you want me to change it to a single LR 87 Model?

Reply:
If ROE has a good one, yes, that would do nicely.